packages/amplify-headless-interface/src/interface/api/add.ts

```diff
@@ -57,7 +57,7 @@ export interface ConflictResolution {
   /**
    * Strategies that will be used for individual models.
    */
-  perModelResolutionStrategy?: PerModelResolutionstrategy[];
+  perModelResolutionstrategy?: PerModelResolutionstrategy[];
```

Comment:
capitalize strategy

Reply:
I can't capitalize this because I need to get alignment with data team since this is an api change. Will follow up in a separate PR